test/language/expressions/object/method-definition/early-errors-object-method-eval-in-formal-parameters.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+esid: pending
+description: It is a SyntaxError if FormalParameters contains eval in strict mode
+negative: SyntaxError
```

Comment:
same thing here --- this restriction is only enforced in strict mode, needs [onlyStrict]